config/src/config/storage_config.rs

```diff
@@ -68,18 +69,24 @@ pub struct StoragePrunerConfig {
     /// Maximum version to prune per batch, should not be too large to avoid spike in disk IO caused
     /// by large batches in the pruner.
     pub max_version_to_prune_per_batch: Option<u64>,
+
+    /// Batch size of the versions to be sent to the pruner - this is to avoid slowdown due to
+    /// issuing too many DB calls and batch prune instead.
+    pub pruning_batch_size: usize,
```

Comment:
Do we really need both this and `max_version_to_prune_per_batch` above?

Reply:
Good point, removed this as we should not need with new batching functionality.